EpiAware/test/test_inference-methods.jl

```diff
@@ -0,0 +1,135 @@
+@testitem "Testing _run_manypathfinder function" begin
+    using Turing, Pathfinder
```

Comment:
are you testing the all failure case here?

Reply:
Probably not...

Reply:
I've beefed up the fail case testing